chain/src/main/java/org/ethereum/beacon/chain/storage/BeaconChainStorage.java

```diff
@@ -1,12 +1,16 @@
 package org.ethereum.beacon.chain.storage;
 
+import org.ethereum.beacon.core.BeaconBlockHeader;
+import org.ethereum.beacon.db.source.DataSource;
 import org.ethereum.beacon.db.source.SingleValueSource;
 import tech.pegasys.artemis.ethereum.core.Hash32;
 
 public interface BeaconChainStorage {
 
   BeaconBlockStorage getBlockStorage();
 
+  DataSource<Hash32, BeaconBlockHeader> getBlockHeaderStorage();
```

Comment:
Why not wrap it with dedicated interface like others. Or, lets postpone it?

Reply:
Let's postpone, since I'm not yet sure how this header storage should be organized